docker/import-osm/import_osm.sh

```diff
@@ -25,7 +25,8 @@ function import_pbf() {
         -cachedir "$IMPOSM_CACHE_DIR" \
         -read "$pbf_file" \
         -deployproduction \
-        -write $diff_flag
+        -write $diff_flag \
+        -config "$CONFIG_JSON"
```

Comment:
```suggestion
        ${CONFIG_JSON:+-config "$CONFIG_JSON"}
```

Reply:
I do the same as other imports commands.

https://github.com/openmaptiles/openmaptiles-tools/blob/9068be14e38e942a8b180b868c66e6d34cb8fb9a/docker/import-osm/import_osm.sh#L29

Reply:
@frodrigo I am not sure i understand what other commands you mean.  If there are users of `import_osm.sh`, adding a new argument with a new environment variable will break their code.
We wouldn't want that to happen - so to make it backwards compatible, we should handle
the case when `$CONFIG_JSON` is not set.

Reply:
The default value are set in `Dockerfile` at https://github.com/openmaptiles/openmaptiles-tools/blob/9068be14e38e942a8b180b868c66e6d34cb8fb9a/docker/import-osm/Dockerfile#L51